app/models/dog_searcher.rb

```diff
@@ -71,7 +71,9 @@ def self.search(params: {}, manager: false)
   private
 
   def tracking_id_search?
```

Comment:
👍 makes more sense to check for 'only numbers?' instead of checking for 'any letters?'. I will revise. Thanks!